calico_node/filesystem/etc/calico/confd/config/custom_filters.cfg

```diff
@@ -0,0 +1,5 @@
+# Generated by confd
```

Comment:
This file is a placeholder to not break BIRD if we don't yet have needed paths/dirs in Etcd. So this placeholder will be shipped along with container. But if/when those paths/dirs are created in Etcd, then this file is going to be re-written by Confd.